azure-pipelines.yml

```diff
@@ -468,10 +468,10 @@ jobs:
   displayName: Test Python Wheels
   strategy:
     matrix:
-      Python39:
-        python.version: '3.9'
```

Comment:
We should not drop Python 3.9's test as we need to make sure our code works on Python 3.9